examples/src/nft.rs

```diff
@@ -0,0 +1,67 @@
+use std::path::Path;
+
+use runner::*;
+
+const NFT_WASM_FILEPATH: &str = "./examples/res/non_fungible_token.wasm";
+
+#[runner::main(sandbox)]
+async fn main() {
+    let (contract_id, signer) = dev_deploy(Path::new(NFT_WASM_FILEPATH)).await.unwrap();
+
+    // Wait a few seconds for create account to finalize:
+    // TODO: exponentialBackoff to not need these explicit sleeps
+    tokio::time::sleep(tokio::time::Duration::from_secs(5)).await;
+
+    let outcome = call(
+        &signer,
+        contract_id.clone(),
+        contract_id.clone(),
+        "new_default_meta".to_string(),
+        format!("{{\"owner_id\": \"{}\"}}", contract_id).into(),
+        None,
+    )
+    .await
+    .unwrap();
+    println!("new_default_meta outcome: {:#?}", outcome);
+
+    tokio::time::sleep(tokio::time::Duration::from_secs(5)).await;
+    let deposit = 10000000000000000000000;
+    let outcome = call(
+        &signer,
+        contract_id.clone(),
+        contract_id.clone(),
+        "nft_mint".to_string(),
+        format!(
```

Comment:
I thinking using serde's `json!` macro makes this easier to read and has the benefit of pulling in value in the current scope.

Reply:
done